bin/mozilla/custom_am.pl

```diff
@@ -635,11 +647,11 @@ sub do_dbcheck {
             - round(amount::numeric,2) diff
     FROM ap
     WHERE
-        (SELECT ROUND(SUM(ac.amount)::numeric,2)
+        ABS((SELECT ROUND(SUM(ac.amount)::numeric,2)
         FROM acc_trans ac
         JOIN chart c ON c.id = ac.chart_id
         WHERE ac.trans_id = ap.id AND c.link LIKE '%AP_paid%' )
-        - round(amount::numeric,2) <> 0
+        - round(amount::numeric,2)) > $form->{mindiff}
```

Comment:
do we rlly want that in the ledger doctor? Ledger docotr should show the problematic bookings anyway, shouldn't it?

Reply:
This is just a convenience needed by Sebastian. We can always show all bookings by putting zero in the minimum diff column.

Reply:
ok.